symengine/__init__.py

```diff
@@ -7,7 +7,7 @@
         MatrixBase, Basic, Lambdify, LambdifyCSE, Lambdify as lambdify,
         DictBasic, symarray, series, diff, zeros, eye, diag, ones,
         Derivative, Subs, add, expand, has_symbol, UndefFunction,
-        Function, FunctionSymbol as AppliedUndef)
+        Function, FunctionSymbol as AppliedUndef, true, false)
```

Comment:
You can add the relationals classes like `Eq, Equality` here, ... if they are available in when you do `from sympy import *`